charts/pega/charts/installer/templates/_pega-installer-job.tpl

```diff
@@ -22,6 +22,9 @@ metadata:
 {{ toYaml .root.Values.global.pegaJob.annotations | indent 4 }}
 {{- end }}{{- end }}
   labels:
+  {{- if (eq .root.Values.preventInterruption "true")   }}
+    pdb: "enabled"
+  {{- end }}
```

Comment:
Would this be a generic selector? In the CMC, we are unsure of other deployments that Client may have along with Pega.

Probably selector like `app: installer` would be more specific.

Reply:
Yes the goal/aim here is to have a generic selector applicable for all global.actions.execute types. I have tested with CloudK install SCE, and verified PDB does not prevent job from cleaning up pod on its own, but *does* prevent node from draining while pod is running. If we didn't add new label, by default all job pods have label `job-name: "<job name>"`, but I don't want to conditionalize the selector based on the execute action (thus job name)